core/src/test/scala/org/allenai/common/cache/QueryCacheSpec.scala

```diff
@@ -1,17 +1,27 @@
 package org.allenai.common.cache
 
+import java.io.File
 import org.allenai.common.{ GitVersion, Version, cache }
 import org.allenai.common.testkit.UnitSpec
 import org.scalatest.BeforeAndAfterAll
-import sys.process._
+import spray.json.DefaultJsonProtocol
 import spray.json.DefaultJsonProtocol._
```

Comment:
You shouldn't need this import anymore, since you're now importing from the `FooJsonProtocol`.